src/libs/ajax/workflows-app/Cbas.js

```diff
@@ -11,5 +12,10 @@ export const Cbas = (signal) => ({
       const res = await fetchCbas(cbasUrlRoot)('api/batch/v1/methods', _.mergeAll([authOpts(), jsonBody(payload), { signal, method: 'POST' }]));
       return res.json();
     },
+    getWithoutVersions: async (cbasUrlRoot) => {
+      const keyParams = qs.stringify({ show_versions: false });
+      const res = await fetchCbas(cbasUrlRoot)(`api/batch/v1/methods?${keyParams}`, _.mergeAll([authOpts(), { signal, method: 'GET' }]));
+      return res.json();
+    },
```

Comment:
I think the `show_versions` should be passed as an argument to the function. We should make this method generic  `get`